app/views/admin/bikes/_bike.html.haml

```diff
@@ -111,6 +111,15 @@
                   - if bike.creation_state.bulk_import.ascend?
                     %small
                       Ascend
+          %tr
```

Comment:
The display should be much more prominent when the bike is deleted. And it doesn’t need to show up at all when it is t deleted.